examples/chip-tool-darwin/commands/pairing/PairingCommandBridge.mm

```diff
@@ -0,0 +1,110 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#include "PairingCommandBridge.h"
+#include "../common/CHIPCommandBridge.h"
+#include "PairingDelegateBridge.h"
+#include "platform/PlatformManager.h"
+#include <controller/ExampleOperationalCredentialsIssuer.h>
+#include <crypto/CHIPCryptoPAL.h>
```

Comment:
Are all these includes needed/used (including CryptoPAL, PASESession, etc, etc)?  I would think the framework would handle those bits....

Reply:
Let me remove it.